cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -395,7 +444,7 @@ def data_preparation(
 
     all_cols = CATEGORICAL_COLS + CONTINUOUS_COLS
 
-    # Select features.
+    # Select features
```

Comment:
```suggestion
    # select features
```

Reply:
@SmritiSatyanV, can you similarly convert every code comment's first letter to lowercase?